Added B+E page into office

edwardwilson opened the pull request:
## Description
Refactored office page for cat b+e
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

edwardwilson committed:
Added B+E page into office

rossfauldsbjss commented on src/pages/office/cat-be/office.cat-be.page.ts:
```diff
@@ -0,0 +1,661 @@
+  selector: 'page-office',
+  templateUrl: 'office.cat-be.page.html',
+})
+export class OfficeCatBEPage extends PracticeableBasePageComponent {
```
Cat BE pages should be BasePageComponent not PracticeableBasePageComponent. Just needs a couple of things removed, including practice header in the html.

TomBillingtonUK commented on src/pages/office/cat-b/__tests__/office.cat-b.spec.ts:
```diff
@@ -15,68 +15,68 @@ import {
   PlatformMock,
```
missing page in the file name

ammarhaiderbjss reviewed (approved):
Looks good provided everything compiles!

TomBillingtonUK commented on src/pages/office/cat-b/office.cat-b.page.ts:
```diff
@@ -170,9 +170,9 @@ interface OfficePageState {
 @Component({
   selector: 'page-office',
-  templateUrl: 'office.html',
+  templateUrl: 'office.cat-b.page.html',
```
you've not updated the selector in the scss file for the page

TomBillingtonUK commented on src/pages/office/cat-be/office.cat-be.page.ts:
```diff
@@ -0,0 +1,661 @@
+
+@IonicPage()
+@Component({
+  selector: 'page-office',
```
this selector needs to be updated and scss file updated to match the selector

rossfauldsbjss reviewed (approved):
Looks good, assuming Tom's  comments are followed.

TomBillingtonUK reviewed (approved):
Looks good, a couple of things which you've missed and need to be changed

edwardwilson committed:
PR comment updates

Removed practiceable base page as BE doesn't support it
Renamed selectors
Fixed spec file naming

edwardwilson committed:
Fixed unit test from base component refactor
